ecosystem/sep-0006.md

```diff
@@ -59,7 +59,11 @@ The JWT should be included in all requests as request header:
 Authorization: Bearer <JWT>
 ```
 
-It is highly recommended to require authentication for all endpoints other than `/info` (and optionally `/fee`).
+It is highly recommended to require authentication for all endpoints other than `/info` (and optionally `/fee`). Some functionality, including the [Shared Accounts](#shared-accounts) functionality described in the section below, may only be possible through the use of SEP-10.
+
+### Shared Accounts
```

Comment:
We've been referring to these type of accounts as "Pooled accounts". Some partners prefer the term "Ombinbus Accounts". Is there any specific reason you went with "Shared"?

Reply:
"Shared" is a non-technical and accurate description of what the accounts are. "Pooled" is common in the engineering world (thread pools, worker pool, etc.) but the term isn't commonly used outside it. "Omnibus" is common in the financial world but I never heard that term until I got on calls with payment services.

I'm open to discuss the naming, but I think "shared" is going to communicate what it means to the most about of people.

Reply:
Omnibus accounts is the term closest to the term used in the real world both in securities for trading accounts that hold others funds, and in payments by payment facilitiators who are holding funds for merchants who don't have their own MIDs and accounts, so that term is pretty spot on really.

If I had to choose between shared or pooled, shared would be preferred. Pooled gives me _mixed_ feelings.

I think I like Omnibus more though. It aligns with the real world?

Reply:
I'm not convinced using "omnibus" as the primary term is the right approach. If you look up definitions, the term implies a set of use cases for an account, whereas the term "shared" describes the account's nature, not necessarily what it's used for.

For example, the [investopedia](https://www.investopedia.com/terms/o/omnibusaccount.asp) definition focuses very much on the privacy of entities whose funds are within the account. This is not a goal of changes we're making here.

I would rather avoid conflating the two subject areas, but we could mention the similarities in the `Shared Accounts` section?

Reply:
Shared is simple and easy to understand, I upvote for Shared.

Reply:
The description on investopedia is very use case specific, but in my experience the term is used broadly to refer to an account that is holding funds for multiple people where the funds are segregated.

Interestingly, in MasterCard documentation they refer to the account holding shared funds an "omnibus, segregated trust, or **pooled account**." It would seem the term pooled account is used in literature and not uncommon as I thought.

One thought about shared is that the term is so generic it could be confused with use cases that are not compatible with it. For example, with a joint or multi-person account where funds are not segregated within the account, we would say that is not a "shared account" which doesn't make sense, because it is. The term is so generic it also may not be easy to search for. For example, if an implementer is trying to find best practices about accounts used to pool multiple people's accounts, searching "stellar shared accounts" may not be fruitful.

I'd lean towards omnibus because it is a well understood term in financial products, it is unambiguous as to the meaning, it is a unique term easy to search for. I think shared and pooled are also okay. Pooled is maybe a little more specific than shared is.

Reply:
What if we tool a similar approach to MasterCard and listed applicable terms? I think only using the term omnibus is going to cause some folks to not realize that section is relevant to them.

```
## Shared, Omnibus, or Pooled Accounts

...
```

Reply:
Not a bad idea. I like. This could be a good way to ensure we zero folks in who are coming from different backgrounds. We can then still pick one term to refer to them in that section.